icu4j/main/tests/framework/src/com/ibm/icu/dev/test/UnicodeKnownIssues.java

```diff
@@ -0,0 +1,151 @@
+package com.ibm.icu.dev.test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.Map.Entry;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * “Known issues” manager.
+ * Intended to be shared between ICU, CLDR, &c.
+ * Test frameworks can create an instance of this to manage known issues
+ */
+public class UnicodeKnownIssues {
```

Comment:
I didn't look at this here. I assume it's copied from CLDR. Please let me know if it's different.

Reply:
yes copied